client/src/global/components/atomic/Button/index.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as Styled from "./styles";
+
+export default function Button({
+  children,
+  size = "sm",
+  secondary,
+  dark,
+  icon,
+  iconSize,
+  width,
+  ...props
+}) {
+  return (
+    <Styled.Button
+      $secondary={secondary}
+      $size={size}
+      $darkMode={dark}
+      $width={width}
+      {...props}
+    >
+      {icon && <Styled.ButtonIcon icon={icon} size={iconSize} />}
+      {children && <Styled.ButtonText>{children}</Styled.ButtonText>}
+    </Styled.Button>
+  );
+}
```

Comment:
There's some work to do on this, I think, to make it a bit more "global", to accommodate the nuances that appear with in other parts of the application. I'd like to give it some more thought and I can take a shot or we can discuss it together. For the sake of velocity, though, let's leave this as-is for now and revisit it later.

Reply:
Totally. I didn't try to accommodate any button styles that aren't used in the heroes. I suspect there's enough nuance that we'll want to do a something like a `BaseButton` and patterns, sort of like what Jen has on NGLP, rather than adding a bunch of props to control styling. Happy to let you do the next step or I can come back to it at some point after we discuss.